lib/redisStore.js

```diff
@@ -0,0 +1,53 @@
+import { REDIS_KEY_TTL_IS_NOT_SET } from './constants.js'
+
+const createKey = (key, namespace) => {
```

Comment:
IMO this could be inside the `RedisStore` class, so you can specify just the key

Reply:
I put it outside of the class because it's only an helper function and I wanted it to be private